### PR TITLE
new interview questions

### PR DESCRIPTION
**Pull request by griffind2009**
added new questions under general section and JS section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,8 +155,13 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is event loop?
   * What is the difference between call stack and task queue?
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
+* What is object-oriented JavaScript (OOJS)?
+* What are teh 6 primitive data types in JavaSript?
```
Comment (vvscode) [inline README.md]: not sure if we need to give hint about `6`

**Comment by vvscode** [comment]
My only question - maybe it worth to ask about data-types in general? not only about primitive? 

There nearby questions like `what is the type for functions` ? `how to detect an array?` `how to detect type?`

File: README.md (hunk 1 above)
Comment (griffind2009) [reply README.md]: ok, should I take out the # 6?

**Comment by griffind2009** [comment]
Good point. Should I take out the question and change it to something more specific? Such as, "Explain the difference between null and undefined.", "What is a boolean?"

File: README.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ This file contains a number of front-end interview questions that can be used wh
 * Explain what ARIA and screenreaders are, and how to make a website accessible.
 * Explain some of the pros and cons for CSS animations versus JavaScript animations.
 * What does CORS stand for and what issue does it address?
+* What is the difference between merging vs. rebasing?
```
Comment (blackfalcon) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ This file contains a number of front-end interview questions that can be used wh
...
+* How do you stay up to date on technology and the what is happening in the technology industry?
```
Comment (blackfalcon) [inline README.md]: This feels like an alternative to, or a follow up of `What did you learn yesterday/this week?`

If this were to be added, I'd suggest putting it directly under the previously stated question.

File: README.md [hunk 4]
```diff
@@ -153,8 +155,13 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is event loop?
   * What is the difference between call stack and task queue?
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
+* What is object-oriented JavaScript (OOJS)?
```
Comment (blackfalcon) [inline README.md]: This question is confusing to me as JavaScript is not specifically an object-oriented language. 

> JavaScript, one of the world's most widely distributed languages, has the properties of an untyped functional language, in addition to imperative and object-oriented paradigms.

I'd suggest something more straight forward to test a person's understanding of the language paradigms.

```
Can you explain or illustrate the differences between object-oriented and functional paradigms in JavaScript?
```

**Pull request closed without merging**